src/UCTSearch.cpp

```diff
@@ -108,7 +108,7 @@ void UCTSearch::dump_stats(KoState & state, UCTNode & parent) {
     const int color = state.get_to_move();
 
     // sort children, put best move on top
-    m_root.sort_root_children(color);
+    parent.sort_root_children(color);
```

Comment:
I think sort_root_children is now safe to use anywhere, so it should probably just be renamed.

Reply:
renamed.